### PR TITLE
Edits to UG and DG (2nd revision)

### PR DESCRIPTION
**Pull request by Bisceto**
Fixed table of contents, edited language and grammatical errors.

### CODE REVIEW

**Review by Jnjy — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -41,54 +41,54 @@ license, making it open source and available for anyone to use and modify.
     - [Create a new tag](#create-a-new-tag)
     - [Tag a `Food`](#tag-a-food)
     - [Untag a `Food`](#untag-a-food)
-    - [List `Food` by tag(s).](#list-foods-by-tags)
-    - [Delete `Food` by tag(s).](#delete-foods-by-tags)
+    - [List `Food` by tag(s).](#list-food-by-tags)
+    - [Delete `Food` by tag(s).](#delete-food-by-tags)
```
Comment (Jnjy) [inline docs/DeveloperGuide.md:45]: Is it okay to remove the fullstop at the end of the description? Same for View Food in table of content.

**Review by Jnjy — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Bisceto) [reply docs/DeveloperGuide.md:45]: Resolved!

**Review by Jnjy — APPROVED** [review]
LGTM

**Pull request merged**